Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "Apache-2.0"
 authors = ["LINKS Foundation"]
 description = "Rust crypto library for zero-knowledge proofs"
 repository = "https://github.com/Cybersecurity-LINKS/zkryptium"
-rust-version = "1.65"
+rust-version = "1.66"
```

Comment:
If you don't think the msrv can be bumped, it would be a pretty small change to fix. Let me know!